platform/tss/tss_message_map_key.proto

```diff
@@ -28,23 +30,31 @@ package com.hedera.hapi.platform.tss;
  * limitations under the License.
  */
 
-option java_package = "com.hedera.hapi.platform.tss.legacy";
-// <<<pbj.java_package = "com.hedera.hapi.platform.tss">>> This comment is special code for setting PBJ Compiler java package
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.state.tss">>> This comment is special code for setting PBJ Compiler java package
```

Comment:
These also should not change.

Reply:
By 'these' do you mean both the `java_package` on line 33 _and_ the `pbj.java_package` on line 34? 
(I'm puzzled why we already have `...tss.legacy` packages for TSS protos)

Reply:
Yes, I mean all of the package statements.

The `.tss.legacy` choice was part of a fairly long discussion and arose because PBJ has a bug (several really) that requires setting the (typically optional) `java_package`, requires it be _different_ from the `pbj.java_package`, and also requires `pbj.java_package` be set (even if the `package` directive correctly sets the java package desired).